src/riak_cs_wm_utils.erl

```diff
@@ -348,6 +349,20 @@ streaming_get(RcPool, RcPid, FsmPid, StartTime, UserName, BFile_str) ->
             {Chunk, fun() -> streaming_get(RcPool, RcPid, FsmPid, StartTime, UserName, BFile_str) end}
     end.
 
+-spec lower_case_method(atom() | string()) -> atom().
+lower_case_method('GET') -> get;
+lower_case_method('HEAD') -> head;
+lower_case_method('POST') -> post;
+lower_case_method('PUT') -> put;
+lower_case_method('DELETE') -> delete;
+lower_case_method('TRACE') -> trace;
+lower_case_method('CONNECT') -> connect;
+lower_case_method('OPTIONS') -> options;
+lower_case_method(Method) when is_atom(Method) ->
```

Comment:
Do we have to support other methods than these? It looks like nobody reaches here unless we make `known_methods`:

```erlang
default(known_methods) ->
    ['GET', 'HEAD', 'POST', 'PUT', 'DELETE', 'TRACE', 'CONNECT', 'OPTIONS'];
```
 https://github.com/basho/webmachine/blob/develop/src/webmachine_resource.erl#L59